virtusize-ui/src/main/java/com/virtusize/android/ui/avatar/VirtusizeAvatarSize.kt

```diff
@@ -0,0 +1,56 @@
+package com.virtusize.android.ui.avatar
+
+import android.content.Context
+import com.virtusize.android.ui.R
+
+enum class VirtusizeAvatarSize {
+    SMALLER {
+        override fun getAvatarSize(context: Context): Float {
+            return context.resources.getDimension(R.dimen.vs_avatar_smaller_size)
+        }
+
+        override fun getAvatarRadiusSize(context: Context): Float {
+            return context.resources.getDimension(R.dimen.vs_avatar_smaller_size) / 2
```

Comment:
There is no way to have a common function which divid by 2 ? Then pass the value as param ?

Reply:
It can reuse getAvatarSize and divide the value by 2
[fe7c600](https://github.com/virtusize/integration_android/pull/91/commits/fe7c60027edabe0f8e80154351d605133359050b)

Reply:
Verified thanks.